misk-redis/src/main/kotlin/misk/redis/RealRedis.kt

```diff
@@ -357,10 +357,22 @@ class RealRedis(
   // pipelined() returns the jedis to the pool, despite returning a Pipeline that holds a reference
   // to the borrowed jedis connection.
   // This is a bug, and will be fixed in a follow-up.
+  @Deprecated("Use pipelining instead.")
   override fun pipelined(): Pipeline {
     return unifiedJedis.pipelined() as Pipeline
   }
```

Comment:
This is actually just entirely broken for clustered redis and will result in a class cast exception if you attempt to use it as is.